debian/autobake-deb.sh

```diff
@@ -79,6 +79,18 @@ then
   sed '/libzstd-dev/d' -i debian/control
 fi
 
+# If openjdk-11-jre is not available (before Debian Buster and Ubuntu Bionic)
+# use an older package that also provides libjawt.so and libjvm.so which are
+# required to build JdbcInterface.jar for the Connect storage engine
+# openjdk-8-jre-headless: libjawt.so and libjvm.so
+# openjdk-11-jre: libjawt.so
+# openjdk-11-jre-headless: libjvm.so
+if ! apt-cache madison openjdk-11-jre | grep 'openjdk-11-jre' >/dev/null 2>&1
+then
```

Comment:
CMake file said that it supports `Java 1.6`, I think here is 6 needed instead of 8 as minimum version.

Reply:
Ok.